config/migrations/2022/20220829135928-erediesten-data/20220928100842-adapt-end-dates-of-worship-organen.sparql

```diff
@@ -0,0 +1,54 @@
+
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+
+DELETE {
+  GRAPH ?g {
+    ?bestuursorgaanInTime mandaat:bindingEinde "2026-04-01T00:00:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
+  }
+} INSERT {
+  GRAPH ?g {
+    ?bestuursorgaanInTime mandaat:bindingEinde "2026-03-31T00:00:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
```

Comment:
@cecemel Ah dammit, mmmh not sure because 00:00:00 is midnight, so belgian timezone would be `03-31T02:00:00` no ? So still 31/03 ? We'd have an issue if we would have put it at 23:59:59

Reply:
Aha, problem solved, it should be a date and not a datetime. I'll correct the migration !

```
(define-resource bestuursorgaan ()
  :class (s-prefix "besluit:Bestuursorgaan")
  :properties `((:naam :string ,(s-prefix "skos:prefLabel"))
                (:binding-einde :date ,(s-prefix "mandaat:bindingEinde"))
                (:binding-start :date ,(s-prefix "mandaat:bindingStart")))
```

Reply:
@claire-lovisa can you check in the data and in the model? I am not entirely sure the API is correct....

Reply:
As discussed in the chat, we use an outdated model.

![image](https://user-images.githubusercontent.com/23281659/193527526-0fc5e650-5df4-47b1-b912-2f3063014eee.png)

So we'll stay with dates in Loket and datetimes in OP for now, knowing we'll have to fix this later on.